devnet.env

```diff
@@ -0,0 +1,13 @@
+OBSERVER_PUBKEY="PUBLIC_KEY_FROM_KEY_FILE"
+OBSERVED_SHARD=0
+GENESIS_BLOCK=b099dbc444824f5bc30bf194a071f43c8e2019ae6213a641f12974fa6a6db5fb
```

Comment:
Chosen as `prevBlockHash` of https://devnet-gateway.elrond.com/block/0/by-nonce/1